opossum-tool/OpossumUI
# Locate popup styling changes

### Summary of changes
Small styling changes.

### Context and reason for change
Fix #1968

### How can the changes be tested
Open the app and check the popup
![image](https://github.com/opossum-tool/OpossumUI/assets/85183359/36277082-2273-4256-a437-f2cb0b1643f9)
